### DISCUSSION TITLE
country fiche 2024

### DISCUSSION THREAD

**Discussion by oberseri** [INSPIRE-MIF/helpdesk#189] [Q&A] [answered]
Dear INSPIRE - team,

could you provide us with information on how to update the country fiche this year.
The relevant links in the current guidance (from 2019) do not work any more - they are redirected to the new homepage (INSPIRE knowledge base).

Thanks and best regards
Erik

[INSPIRE_ReportingGuidance2019_v1.0.pdf](https://github.com/INSPIRE-MIF/helpdesk/files/14404448/INSPIRE_ReportingGuidance2019_v1.0.pdf)

**Comment by fabiovinci** [comment]
Dear @oberseri,

the new procedure for the delivery of country fiches is undergoing final revision.

You will receive more information soon.

**Comment by MarcoMinghini** [comment, marked as answer]
Dear @oberseri,
starting this year, country fiches should be directly updated in this GitHub repository: https://github.com/INSPIRE-MIF/INSPIRE-in-your-Country.

Please have a look at the README and in particular to the instructions available [here](https://github.com/INSPIRE-MIF/INSPIRE-in-your-Country/tree/main/how-to-report-new-CF). In case you encounter any problem, please report it using the [issue tracker](https://github.com/INSPIRE-MIF/INSPIRE-in-your-Country/issues) of this new repository.
